### PR TITLE
External_Id field, incorrect type long? converted to string

### PR DESCRIPTION
**Pull request by acolom**
Hi,

this is my first pull request, i'm newbie in GItHub and I hope that i've done things correctly.

I found this problem that causes newtonsoft to throw a serialization exception in case External_Id is setted to a non long value (a string).

As Zendesk's api says [zendesk] (https://developer.zendesk.com/rest_api/docs/core/tickets#content) external id is a string not a long. 

I've just modified it, the types of the properties.

### CODE REVIEW

File: src/ZendeskApi.Contracts/Models/Organization.cs [hunk 1]
```diff
@@ -49,8 +49,8 @@ public class Organization : IZendeskEntity
         [IgnoreDataMember]
         public long group_id { get; set; }
 
-        [IgnoreDataMember]
-        public long external_id { get; set; }
+        [DataMember(Name = "external_id", EmitDefaultValue = false)]
```
Comment (hjaltethor) [inline src/ZendeskApi.Contracts/Models/Organization.cs]: Hi! Thank you for your pull request. Could I get you to move this field up to the other non-ignored fields? It helps the readability of the class.

Comment (acolom) [inline src/ZendeskApi.Contracts/Models/Organization.cs]: Hello,

i've just commited this change to my branch. As i told you i'm totally a newbie on Github and Git, i synched my changes to my repository. i need to do something else?

Comment (hjaltethor) [inline src/ZendeskApi.Contracts/Models/Organization.cs]: If you make the change in the same branch as the one you created this PR from (fix-external-id-type) and commit the change, then it should be automatically added here.

Comment (acolom) [inline src/ZendeskApi.Contracts/Models/Organization.cs]: I beleive that i did it

**Comment by hjaltethor** [comment]
This has been released and should show up in the latest version on nuget.

**Pull request merged**